src/maintenance/azext_maintenance/generated/_params.py

```diff
@@ -23,188 +23,219 @@
 def load_arguments(self, _):
 
     with self.argument_context('maintenance public-configuration show') as c:
-        c.argument('resource_name', type=str, help='Resource Identifier', id_part='name')
+        c.argument('resource_name', type=str, help='Maintenance Configuration Name. Swagger name=resourceName',
+                   id_part='name')
 
     with self.argument_context('maintenance applyupdate show') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('provider_name', type=str, help='Resource provider name')
-        c.argument('resource_type', type=str, help='Resource type')
-        c.argument('resource_name', type=str, help='Resource identifier')
-        c.argument('apply_update_name', options_list=['--name', '-n', '--apply-update-name'], type=str, help=''
-                   'applyUpdate Id')
+        c.argument('provider_name', type=str, help='Resource provider name. Swagger name=providerName')
+        c.argument('resource_type', type=str, help='Resource type. Swagger name=resourceType')
+        c.argument('resource_name', type=str, help='Resource identifier. Swagger name=resourceName')
+        c.argument('apply_update_name', options_list=['--name', '-n', '--apply-update-name'], type=str,
+                   help='applyUpdate Id. Swagger name=applyUpdateName')
```

Comment:
@changlong-liu and @qiaozha Why does help message contain "Swagger name=*"?

Reply:
Code is autogenerated from autorest CLI

Reply:
Looks like it due to --debug flag provided to autorest command.
--debug flag should not change behavior of code generation
 
 
https://github.com/Azure/autorest.az/blob/c85ea6a31e4dd7e45b4de0762fcc018b0062f6f6/src/generate/renders/generated/CliParams.ts#L416

Reply:
I have regenerated code without --debug flag.

Please review. PR latest iteration.